--- local-doh.go
+++ local-doh.go
@@ -25,7 +25,6 @@
 	}
 	defer proxy.clientsCountDec()
 	dataType := "application/dns-message"
-	writer.Header().Set("Server", "dnscrypt-proxy")
 	if request.URL.Path != proxy.localDoHPath {
 		writer.WriteHeader(404)
 		return
@@ -33,7 +32,7 @@
 	if request.Header.Get("Content-Type") != dataType {
 		writer.Header().Set("Content-Type", "text/plain")
 		writer.WriteHeader(400)
-		writer.Write([]byte("dnscrypt-proxy local DoH server\n"))
+		writer.Write([]byte("It Works!\n"))
 		return
 	}
 	start := time.Now()
